src/components/stepper/index.js

```diff
@@ -4,9 +4,15 @@ import Button from "../button";
 import "./style.css";
 import { useState } from "react";
 
-const Stepper = ({ header, children, onComplete }) => {
+const Stepper = ({ header, children, onComplete, nextBtnDisabled }) => {
     const [currentStep, setCurrentStep] = useState(0)
 
+    let buttonDisabled = false
```

Comment:
You can bypass this unnecessary code by setting `nextBtnDisabled = false`

Reply:
The reason for this statement is to make it easier to use, if this component gets reused but the value of `nextBtnDisabled` is not set it will default to false, but if `nextBtnDisabled` is set to be a certain value it will use that specified value.